### PR TITLE
Try to fix CircleCI again.

### PR DESCRIPTION
**Pull request by fatlotus**
Trial and error, the best debugging method.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -9,14 +9,9 @@ test:
...
 deployment:
-  development:
-    branch: /(?!master)/
+  app_engine:
+    branch: /.*/
     commands:
       - rm venv
       - ~/google_appengine/appcfg.py update . -A caravel-code-reviews --version=$(git rev-parse --abbrev-ref HEAD) --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN
-
-  production:
```
Comment (georgeteo) [inline circle.yml]: Will this still deploy the master to hosted-caravel.appspot.com?

Comment (fatlotus) [inline circle.yml]: Good eyes. Fixed.

File: circle.yml [hunk 2]
```diff
@@ -9,14 +9,9 @@ test:
...
-    branch: master
-    commands:
-      - rm venv
-      - ~/google_appengine/appcfg.py update . --version=master --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN
+      - if [ "x$(git rev-parse --abbrev-ref HEAD)" = "xmaster" ]; then ~/google_appengine/appcfg.py update . --version=$(git rev-parse --abbrev-ref HEAD) --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN; fi
```
Comment (georgeteo) [inline circle.yml]: Cool. Also, why xmaster and x$(git rev-parse --abbrev-ref HEAD)? (Why x?)

Comment (fatlotus) [inline circle.yml]: Ehhhh, it's a weird habit. If "git rev-parse --abbrev-ref HEAD" is empty, then without the x there'd be a syntax error. That shouldn't ever be the case, so I've disabled it.

**Pull request merged**